tests/scalers/elasticsearch.test.ts

```diff
@@ -0,0 +1,329 @@
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+import {waitForRollout} from "./helpers";
+
+const testNamespace = 'elasticsearch-test'
+const elasticsearchNamespace = 'elasticsearch'
+const deploymentName = 'podinfo'
+const indexName = 'keda'
+const searchTemplateName = 'keda-search-template'
+const elasticPassword = 'passw0rd!'
+const kubectlExecCurl = `kubectl exec -n ${elasticsearchNamespace} elasticsearch-0 -- curl -sS -H "content-type: application/json" -u "elastic:${elasticPassword}"`
+
+test.before(t => {
+    // install elasticsearch
+    sh.exec(`kubectl create namespace ${elasticsearchNamespace}`)
+    const elasticsearchTmpFile = tmp.fileSync()
+    fs.writeFileSync(elasticsearchTmpFile.name, elasticsearchStatefulsetYaml.replace('{{ELASTIC_PASSWORD}}', elasticPassword))
+
+    t.is(0, sh.exec(`kubectl apply --namespace ${elasticsearchNamespace} -f ${elasticsearchTmpFile.name}`).code, 'creating an elasticsearch statefulset should work.')
+    t.is(0, waitForRollout('statefulset', "elasticsearch", elasticsearchNamespace))
```

Comment:
I think e2e tests are failing because this could need a bit more time. WDYT?
```suggestion
    t.is(0, waitForRollout('statefulset', "elasticsearch", elasticsearchNamespace, 300))
```

Reply:
the elasticsearch image is huge (~1.2GB) and on my 6 cores laptop it takes ~20-30 seconds to start.
Let's check with 300 seconds

Reply:
I will run again e2e tests when you increase the  waiting timeout :)